Assets/Scripts/Tutorial/TutorialController.cs

```diff
@@ -68,9 +68,9 @@ private void OnEnable()
         }
         private void Setup()
         {
-            _notebook.ShowChapter(Chapter.ChapterTitle.Tutorial);                         // display the tutorial chapter
-            _notebook.CurrentChapter.PresentPage(_notebook.CurrentChapter.defaultPage);   // display the default page of the tutorial chapter
+            _notebook.CurrentChapter.Setup();
```

Comment:
wait sorry it's better if here you do _notebook.ShowChapter(Chapter.ChapterTitle.Tutorial, false, true) to not reset the current chapter and reset the chapter you want to present (the tutorial chapter)